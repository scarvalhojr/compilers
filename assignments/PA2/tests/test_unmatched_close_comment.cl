*)
CLASS Foo {
    value : Int <- 0;
}
