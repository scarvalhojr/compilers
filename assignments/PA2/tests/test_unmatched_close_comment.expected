#name "tests/test_unmatched_close_comment.cl"
#1 ERROR "Unmatched *)"
#2 CLASS
#2 TYPEID Foo
#2 '{'
#3 OBJECTID value
#3 ':'
#3 TYPEID Int
#3 ASSIGN
#3 INT_CONST 0
#3 ';'
#4 '}'
